### PR TITLE
Gallery listings2

### PR DESCRIPTION
**Pull request by johnpooler**

### CODE REVIEW

**Comment by RafSobolQAC** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- app/services/UpcomingServices.scala  1
- app/controllers/UpcomingController.scala  1
- app/controllers/ReleasedController.scala  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5257596&bid=17226137)

File: app/controllers/ReleasedController.scala [hunk 1]
```diff
@@ -19,7 +19,9 @@ class ReleasedController @Inject()(
 
   def getMovies = Action.async { implicit request: Request[AnyContent] =>
     releasedServices.getMovies.map { movies =>
-      Ok(movies.toString())
+//   Ok(movies.toString())
```
Comment (nathan-rees) [inline app/controllers/ReleasedController.scala:22]: mb remove this line

File: app/models/Movie.scala [hunk 2]
```diff
@@ -2,21 +2,48 @@ package models
...
+//{
+//  def getId(): Int = _id
+//  def getTitle(): String = title
+//  def getDescription(): String = description
+//  def getPosterUrl(): String = image
+//  def getActors(): List[String] = actors
+//  def getDirector(): String = director
+//  def getScreenings():List[LocalDateTime] = screenings
+//
+//}
```
Comment (nathan-rees) [inline app/models/Movie.scala:46]: remove meeeeeee

**Review by nathan-rees — APPROVED** [review]

**Comment by RafSobolQAC** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- app/controllers/UpcomingController.scala  1
- app/controllers/ReleasedController.scala  1
- app/services/UpcomingServices.scala  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5257596&bid=17226137)

**Pull request merged**